src/elements/content-sidebar/__tests__/ActivitySidebar.test.js

```diff
@@ -883,11 +884,141 @@ describe('elements/content-sidebar/ActivitySidebar', () => {
+            ${'open'}     | ${getOpenItems}
+            ${'resolved'} | ${getResolvedItems}
+        `(
+            'should filter feed items of type "comment" or "annotation" based on status equal to $status',
+            ({ status, getExpected }) => {
+                instance.setState({
+                    feedItemsStatusFilter: status,
+                });
+                expect(instance.getFilteredFeedItems()).toMatchObject(getExpected());
+            },
+        );
+    });
+
+    describe('handleItemsFiltered()', () => {
+        let instance;
+        let wrapper;
+
+        beforeEach(() => {
+            wrapper = getWrapper();
+            instance = wrapper.instance();
+            instance.setState = jest.fn();
+        });
+
+        test.each`
+            status        | expected
+            ${undefined}  | ${undefined}
+            ${'open'}     | ${'open'}
+            ${'resolved'} | ${'resolved'}
+        `('given $status should update feedItemsStatusFilter state with $expected', ({ status, expected }) => {
+            instance.handleItemsFiltered(status);
+            expect(instance.setState).toBeCalledWith({ feedItemsStatusFilter: expected });
+        });
+    });
+
     describe('renderAddTaskButton()', () => {
         test('should return null when hasTasks is false', () => {
             const wrapper = getWrapper({ hasTasks: false });
             const instance = wrapper.instance();
             expect(instance.renderAddTaskButton()).toBe(null);
         });
     });
+
+    describe('renderActivitySidebarFilter()', () => {
+        describe('should return null', () => {
+            test('when activityFeed.filter feature is not set', () => {
+                const wrapper = getWrapper();
+                const instance = wrapper.instance();
+                expect(instance.renderActivitySidebarFilter()).toBe(null);
+            });
+
+            test('when activityFeed.filter feature is disabled', () => {
+                const wrapper = getWrapper({
+                    features: {
+                        activityFeed: {
+                            filter: { enabled: false },
+                        },
+                    },
+                });
+                const instance = wrapper.instance();
+                expect(instance.renderActivitySidebarFilter()).toBe(null);
+            });
+        });
+
+        test('should return ActivitySidebarFilter when activityFeed.filter feature is enabled', () => {
+            const wrapper = getWrapper({
+                features: {
+                    activityFeed: {
+                        filter: { enabled: true },
+                    },
+                },
+            });
+            const instance = wrapper.instance();
+            const resultWrapper = mount(instance.renderActivitySidebarFilter());
+            expect(resultWrapper.name()).toBe('ActivitySidebarFilter');
+        });
+    });
+
+    describe('renderTitle()', () => {
```

Comment:
Should these be separate components?

Reply:
As discussed, since this is not blocking merging these changes, I would leave it for now as setting `title` and `actions` props of the `SidebarContent` will most likely need more work before integrating threads in Preview.